src/components/navbar.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+const Navbar = () =>{
+    return(
+        <$Navbar>
+                <$NavLogo>
+                    <img src="/images/logo.png" alt="logo" />
+                </$NavLogo>
+                <$NavItem>
+                    <$NavLink href='/leaderboard'>LEADERBOARD</$NavLink>
+                    <$NavLink href='/tips-tricks'>TIPS & TRICKS</$NavLink>
+                    <$NavLink href='/profile'><img src="/images/profile.png" alt="" /></$NavLink>
+                </$NavItem>
+        </$Navbar>
+    )
+}  
+
+const $Navbar = styled.nav`
+    @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Thai:wght@200&display=swap');    
```

Comment:
We shouldn't import here, import in a global css or sumthin

Reply:
ok, i will fix this immediately :-D 💯